backend/core/src/listings/entities/listing.entity.ts

```diff
@@ -664,6 +664,27 @@ class Listing extends BaseEntity {
   @IsDate({ groups: [ValidationsGroupsEnum.default] })
   @Type(() => Date)
   lastApplicationUpdateAt?: Date | null
+
+  @Column({ type: "text", nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  requestedChanges?: string | null
+
+  @Column({ type: "timestamptz", nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsDate({ groups: [ValidationsGroupsEnum.default] })
+  @Type(() => Date)
+  requestedChangesDate?: Date | null
+
+  @ManyToOne(() => User, { eager: true, cascade: true })
```

Comment:
Do we need the eager and cascade here? 
My understanding is that when "eager" is set to true doing a find on listings will then grab the whole user. We probably won't want the user information being sent unless it has to be.
Also, I don't think we will need to update the user table so cascade is probably not necessary. Unless I'm misunderstanding what those two flags do. @YazeedLoonat might know better